vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Tutorial: Install a dependency from a manifest file"
+description: Learn to install your projects dependencies with vcpkg by using a manifest file.
+author: vicroms
+ms.author: viromer
+ms.date: 8/10/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+#CustomerIntent: As a beginner vcpkg user, I want to learn how to install my project dependencies using vcpkg in manifest mode
+---
+# Tutorial: Install a dependency from a manifest file
+
+vcpkg has two operation modes: classic mode and manifest mode. This article
+describes how to install packages using manifest mode, which is the recommended
+workflow for most users.
+
+In manifest mode, you declare your project's direct dependencies in a manifest file named
+[`vcpkg.json`](../reference/vcpkg-json.md).
+
+Manifest files have their own `vcpkg_installed` directory where they install their dependencies,
+unlike classic mode, where all packages are installed in a common `%VCPKG_ROOT%/installed`
+directory. Manifests are suitable for using vcpkg with multiple codebases, as each codebase can have
+a different set of dependencies that do not interfere with each other.
+
+Manifest mode is also required to use advanced features like
+[versioning](../users/versioning.md) and [custom registries](../users/registries.md).
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+> * [1 - Create a project with a manifest]
+> * [2 - Integrate vcpkg with your buildsystem]
+> * [3 - Install the manifest dependencies]
+> * [4 - Build the project]
+
+
+## Prerequisites
+
+* vcpkg
+* A terminal
+* A code editor
+* A C++ compiler
+* (Optional) CMake or MSBuild
+
+## 1 - Create a project with a manifest
+
+In a new folder, create a source file named `main.cxx` with these contents:
+
+:::code language="cxx" source="../examples/snippets/manifest-mode-cmake/main.cxx":::
+
+The code references the open-source libraries: `cxxopts`, `fmt`, and `range-v3`; which are all
+available in the vcpkg public registry at <https://github.com/Microsoft/vcpkg>.
+
+To declare these dependencies, create a file named `vcpkg.json` in the project's folder:
+
+`vcpkg.json`:
+
+:::code language="json" source="../examples/snippets/manifest-mode-cmake/vcpkg.json":::
+
+You only need to specify your direct dependencies in the `"dependencies"` list. When it runs, vcpkg
+resolves and installs any required transitive dependencies.
+
+## 2 - Integrate vcpkg with your buildsystem
+
+In this step we show you how to integrate vcpkg with CMake or MSBuild, so that your project dependencies get
+automatically installed whenever you build the project. If you're using a different build system skip to the
+next step: [3 - Install the manifest dependencies].
```

Comment:
```suggestion
In this step we show you how to integrate vcpkg with CMake or MSBuild, so that your project dependencies get automatically installed or restored whenever you build the project. 

If you're using a different build system, skip to the next step: [3 - Install the manifest dependencies].
```